fares-data-build-tool/create-data
# feat: CFD-360 group links to individuals

# Description

-   To link groups to the individuals within that group so updating the individual will also update the group

# Testing instructions

- Create Individuals in global setting and also create a group
- Change the name of one of the individuals and check that the group is updated with the new name
- Check that the existing journey groups still work and are a distinct list to the global settings groups

# Type of change
